portfolio_home.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'/>
+	<meta name='viewport' content='width=device-width, initial-scale=1.0'/>
+    <title>Jake Rooms - Portfolio</title>
+	<link rel='stylesheet' href='normalize.css'/>
+	<link rel='stylesheet' href='portfolio.css'/>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+</head>
+<body>
+	<main>
+		<!--Html for the top navigation bar-->
+		<nav class='link_bar'>
+			<a class= 'top_link' href='portfolio_aboutme.html'>About me</a>
+		</nav>
+		<!--Html for the image and project navigation bar-->
+		<section class='section_home'>
+			<img class='img_home' src='jake_face.jpg' alt='my face'/>
+			<!--Html for the brojects navigation menu-->
+			<div class='projects'>
+				<h2>Projects</h2>
```

Comment:
this is styled up to look like a button but doesnt have the functionality. Best to make it look more like a title than a button

Reply:
lowered the border radius and removed the outlines